docs/user/03-11-customize_secret.md

```diff
@@ -0,0 +1,45 @@
+# Customize the Default Credentials and Access
+
+You can customize the `sap-btp-manager` Secret and manage your own SAP BTP Operator's default configuration.
+
+## Context
+
+When you create SAP BTP, Kyma runtime, the `sap-btp-manager` Secret is automatically created as the default Secret managing the SAP BTP Operator's resources. 
+Any changes to the Secret are reverted, and the previous settings are restored within up to 24 hours.
```

Comment:
Because the word reconciliation comes up in line 11, I'd introduce it here:

```suggestion
Because of Kyma's automatic reconciliation, any changes to the Secret are reverted, and the previous settings are restored within up to 24 hours.
```
(maybe you can phrase it more elegantly :D)